src/tauri/useTauriFileDrop.ts

```diff
@@ -0,0 +1,43 @@
+import type { Event } from '@tauri-apps/api/event'
```

Comment:
The UI needs the file path to send to the backend, but that's not provided with HTML Drag + Drop. Tauri does provide the file name, but there doesn't seem to be a nice way to only listen for drops on specific elements. It's window-level.

I tried using `onmouse` events, but they don't seem to be fired when a drag operation is happening, at least on macOS.